src/mpid/ch4/shm/posix/eager/include/posix_eager_impl.h

```diff
@@ -26,17 +26,9 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_POSIX_eager_recv_begin(MPIDI_POSIX_eager_recv
     return MPIDI_POSIX_eager_func->recv_begin(transaction);
 }
 
-MPL_STATIC_INLINE_PREFIX void MPIDI_POSIX_eager_recv_memcpy(MPIDI_POSIX_eager_recv_transaction_t *
```

Comment:
This is doing more than just remove `memcpy`. I think it ended up here after rebase

Reply:
Posix used to use this function for copying data into recv buffer. Since it is just a direct memcpy, so I refactored it away by simply calling the copy routines in `MPIDIG_msg.h`.

Reply:
Yes I saw that, so we don't need this anymore and can be deleted.